src/GovUk.Education.ExploreEducationStatistics.Data.Seed/Services/FileStorageService.cs

```diff
@@ -0,0 +1,141 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using static GovUk.Education.ExploreEducationStatistics.Data.Seed.ValidationErrorMessages;
+using static GovUk.Education.ExploreEducationStatistics.Data.Seed.ValidationUtils;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.FileStoragePathUtils;
+namespace GovUk.Education.ExploreEducationStatistics.Data.Seed.Services
+{
+    public class FileStorageService : IFileStorageService
```

Comment:
We should create a ticket to try and abstract a common FileStorageService if we don't already have one.

Reply:
Will do

Reply:
done